src/Helpers/VulnerabilitiesHelper.js

```diff
@@ -11,16 +11,16 @@ export function createCveListByAccount(cveList) {
         let data = payload.data;
         let meta = payload.meta;
 
-        const descriptionMaxLength = 200;
-
         data = data.map(row => ({
             id: row.id,
             cells: [
                 <ImpactBadge impact={row.attributes.impact} hastooltip={true} key={row.id.toString()} />,
                 row.attributes.synopsis,
-                <CVERecordDescription
-                    description={row.attributes.description}
-                    maxstrlength={descriptionMaxLength}
+                <LongTextTooltip
+                    content={row.attributes.description}
+                    maxLength={200}
+                    tooltipPosition={'top'}
+                    tooltipMaxWidth={'50vw'}
                     key={row.id.toString()}
```

Comment:
```suggestion
                     key={row.id.toString()}
                     entryDelay={2000}
```
We need to use higher entry delay as tooltips might get annoying in lists.

Reply:
Will be fixed in other related PR.

Reply:
Please resolve it in this PR - there is no reason not to do it now.